src/mlpack/methods/ann/layer/isrlu.hpp

```diff
@@ -0,0 +1,138 @@
+/**
+ * @file methods/ann/layer/isrlu.hpp
+ * @author Abhinav Anand
+ *
+ * Definition of the ISRLU activation function as described by Jonathan T. Barron.
+ *
+ * For more information, read the following paper.
+ *
+ * @code
+ * @article{
+ *   author  = {Carlile, Brad, Delamarter, Guy, Kinney, Paul, Marti,
+ *              Akiko, Whitney, Brian},
```

Comment:
sorry for the confusion. This was my original suggestion when I thought that these were all last names, as opposed to first and last names.
I'd again recommend a format which makes it clear. Maybe you'd prefer just last names?

```suggestion
 *   author  = {Carlile, Delamarter, Kinney, Marti, Whitney},
```

Reply:
Another way to get first names correctly in BibTeX format would be something like `Carlile, Brad and Delamarter, Guy and Kinney, Paul and Marti, Akiko and Whitney, Brian}`. :+1:

Reply:
^ This whole mess is because I found that format confusing. It must just be me then

Reply:
Yeah, the BibTeX format is confusing but I guess I've gotten used to it.  I think you can also do `Brad Carlile and Guy Delamarter and Paul Kinney and ...`, but I am not 100% sure.  Maybe that's a little better? :smile:

Reply:
I'll add both suggestions, in hope that that's convient for @abh2k 
```suggestion
 *   author  = {Brad Carlile and Guy Delamarter and Paul Kinney and Marti Akiko
 *              and Brian Whitney},
```

Reply:
```suggestion
 *   author  = {Carlile, Brad and Delamarter, Guy and Kinney, Paul and Marti,
 *              Akiko and Whitney, Brian},
```
(Sorry for being confused by this earlier!)